lib/jekyll/scholar/version.rb

```diff
@@ -1,5 +1,9 @@
 module Jekyll
   class Scholar
+<<<<<<< HEAD
```

Comment:
Hey, sorry about that. I missed that one. I had a bunch I need to fix, because I was basically working on my local dev folder which I had tweaked here and there for private use. Sorry. Will be more attentive next time. Glad you liked the new idea.